### DISCUSSION TITLE
Couldn't find manifest for 'dark'.

### DISCUSSION THREAD

**Discussion by wail-shudar** [ScoopInstaller/Scoop#4946] [Help and Questions] [answered]
I'm receiving  

`Couldn't find manifest for 'dark'`

when I run:

`scoop install python38`

I have both the extras and versions buckets.

**Comment by rashil2000** [comment, marked as answer]
You need the main bucket too

**Reply by cryoff** [reply]
Thank you for the hint! I had the same problem, re-installed "main" bucket - and everything worked (interestingly why "main" bucket kinda disappeared..)
